client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -193,6 +193,24 @@ public ShuffleClientImpl(String appUniqueId, CelebornConf conf, UserIdentifier u
             "celeborn-shuffle-split", pushSplitPartitionThreads, 60);
     reviveManager = new ReviveManager(this, conf);
 
+    if (!isDriver) {
+      heartbeater =
+          ThreadUtils.newDaemonSingleThreadScheduledExecutor("lifecycleManager-heartbeater");
```

Comment:
the thread name should start with `celeborn-` as it lives in Spark process

Reply:
done